src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -463,9 +497,10 @@ class XrdAccSciTokens;
 
 XrdAccSciTokens *accSciTokens = nullptr;
 XrdSciTokensHelper *SciTokensHelper = nullptr;
+XrdSciTokensRedir *SciTokensRedir = nullptr;
 
 class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper,
-                        public XrdSciTokensMon
+                        public XrdSciTokensMon, public XrdSciTokensRedir
```

Comment:
This changes the class layout, so as it is it can only go into a major version. @abh3 What do you think?

Reply:
This is the layout of a class in a plugin.  It doesn't affect the ABI of the plugin interface itself.

(This same trick was done with `XrdSciTokensHelper` during the middle of the 5.x series)

Reply:
I believe that technically that is true as this relies on GCC behavior, I believe. So, it may not work for other compilers. However, I would check on that. It may not make much of a difference though as it seems were now running the clock and likely won't be ready for releasing this until 6.0 anyway.

Reply:
I don't understand the comment.

This is a plugin, not an interface.

How could a change in a plugin's implementation cause an ABI/API compatibility break in the interface it implements?